docs/opcodes_details.md

```diff
@@ -217,6 +217,20 @@ see `CALLDATACOPY`, everything is same just the data is from the code not input
 - **evmone**: Data is in `state.host.get_tx_context().block_coinbase`
 - **openethereum**: Data is in `ext.env_info().author.clone()`
 
+## `0x46 CHAINID`
```

Comment:
pls, throw it into the table at `opcodes.csv` and `selection.csv`, you can search the repo by COINBASE and see where we should fit CHAINID in

Reply:
done